### DISCUSSION TITLE
Render adpative card in a hover card

### DISCUSSION THREAD

**Discussion by shia940** [microsoft/fluentui#19134] [General] [answered]
I am trying to render adaptive card on hover card which is attached to some url. In the adaptive card, I have used the Input.ChoiceSet but when I am trying to select or hover on any of input choices, the hover card dismisses. Please can someone help me to fix or is it something not supported?

**Comment by micahgodbolt** [comment, marked as answer]
This is a bit outside of this forum as we don't work much with adapative cards here.
